process/constants.go

```diff
@@ -26,4 +26,5 @@ const (
 
 const ShardBlockFinality = 1
 const MetaBlockFinality = 1
+const BlockFinality = 1
```

Comment:
rename to something with ForkBlockFinality ?

Reply:
Done